src/app/components/Document/index.jsx

```diff
@@ -35,6 +35,14 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
         {title}
         {links}
         {styleTags}
+        {data.isAmp && (
+          <Fragment>
```

Comment:
Have we thought about using the [short syntax](https://reactjs.org/docs/fragments.html#short-syntax) for this yet?

Reply:
That doesn't feel super descriptive. 😕 I'll message around & see how people feel about it.

Reply:
The team discussed this briefly in a Slack channel, & seemed to prefer the explicit option.